### PR TITLE
adds ApacheApp NodeType and implementation

### PR DESCRIPTION
**Pull request by nyuuyn**
Adds ApacheApp to simply use ApacheWebServer

### CODE REVIEW

**Review by lharzenetter — CHANGES_REQUESTED** [review]
"Release" all definitions, i.e., remove the `-wip1`.

**Review by wederbn — APPROVED** [review]
LGTM

File: nodetypes/http%3A%2F%2Fopentosca.org%2Fnodetypes/ApacheApp_w1/NodeType.tosca [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Definitions xmlns="http://docs.oasis-open.org/tosca/ns/2011/12" xmlns:ns0="http://www.opentosca.org/winery/extensions/tosca/2013/02/12" xmlns:selfservice="http://www.eclipse.org/winery/model/selfservice" xmlns:testwineryopentoscaorg="http://test.winery.opentosca.org" targetNamespace="http://opentosca.org/nodetypes" id="nodeTypes-ApacheApp_w1">
+    <NodeType name="ApacheApp_w1" abstract="no" final="no" targetNamespace="http://opentosca.org/nodetypes">
+        <ns0:PropertiesDefinition elementname="properties" namespace="http://opentosca.org/nodetypes/propertiesdefinition/winery">
```
Comment (lharzenetter) [inline nodetypes/http%3A%2F%2Fopentosca.org%2Fnodetypes/ApacheApp_w1/NodeType.tosca:4]: Please also add the parent from which this NT is derived from. I guess `ToscaNormativeNodeTypes:WebApplication`?

Comment (nyuuyn) [reply nodetypes/http%3A%2F%2Fopentosca.org%2Fnodetypes/ApacheApp_w1/NodeType.tosca:4]: added

**Review by lharzenetter — APPROVED** [review]

**Pull request merged**